api/src/main/java/org/openmrs/validator/PersonNameValidator.java

```diff
@@ -164,4 +180,29 @@ private String getFieldKey(String field, boolean arrayInd, boolean testInd) {
 		return testInd ? field : arrayInd ? "names[0]." + field : "name." + field;
 	}
 	
-}
+	
+	private boolean startsOrEndsWithWhitespace(String value) {
+		boolean startsOrEndsWithWhitespace = false;
```

Comment:
I think, here you made more complex for this function. Can we think like this way to reduce the nesting? 

```
private boolean startsOrEndsWithWhitespace(String value) {
   return (StringUtils.isNotBlank(value) && (Character.isSpaceChar(value.charAt(0)) || Character.isSpaceChar(value.charAt(-1)));
}
```